setup.py

```diff
@@ -139,6 +139,7 @@ def initialize_options(self) -> None:
           #
           # https://github.com/aws/aws-cli/blob/develop/setup.py
           'PyYAML<5.3',
+          'mysqlclient',
```

Comment:
Hmmm.  This change is consistent with current practice.  I do agree that the current practice isn't scalable and ought to change.

However, my take is that if we want to change this, we should change it all as a breaking change rather than have different databases handled inconsistently, and someone should take responsibility for migrating internal usage (which will involve testing and adding extras in particular places as needed).  It's unfortunate that this didn't happen during the transition to records-mover, for sure.

I filed an internal ticket for the backlog and tagged you in on it.

Reply:
Hmm, this sounds like all the more reason to put the MySQL dep in an extra now. That means it won’t be a breaking change for users using just MySQL in the future. In addition, isn’t this a breaking change for all current users because they will need to install the MySQL client (the C library) in order to install this?

It does mean inconsistency between the existing stuff and MySQL, but that doesn’t really seem like a problem. That certainly can be fixed at some point in the future when a breaking change is well timed.

Reply:
Well, I guess if you're not flexible on that, then let's go ahead and make the breaking change now.  I'll get another PR out.  It'll be a little more work for teams internally as I'm not sure I can support things as seamlessly as the last breaking change.